### ISSUE TITLE
Add headers and footers including license, sources and the KIT&ScaleIT logo to each page

### ISSUE THREAD

**Issue by frostyandy2k** [ScaleIT-Org/workshop-app-prototyping#1]

**Comment by laurahalbmann** [comment]
Mit each page ist nur das readme gemeint? 

Reicht das für die Sources aus: (further sources: flaticon, https://www.userfocus.co.uk/resources/prototype.html)?

**Comment by frostyandy2k** [comment]
Weiss nicht mehr was damit gemeint war. Definieren wir es neu
* Add source list to the powerpoint (icons, images)
* Add source list to the readme (icons, images)
* Add the KIT and TECO logo to the first slide

**Comment by laurahalbmann** [comment]
[Prototyping Materials_15.03.2018.pptx](https://github.com/ScaleIT-Org/workshop-app-prototyping/files/1815199/Prototyping.Materials_15.03.2018.pptx)

Passt die PPT so? Oder sollen die beiden Logos auf das Readme?

**Comment by frostyandy2k** [comment]
Ja, am besten bei so sachen einen Screenshot in das issue einfügen

**Comment by frostyandy2k** [comment]
Ach warte, ich hab grad das ticket gelesen. 
Kannst du das als footer einsetzen auf jeder Seite?

© ScaleIT, KIT TECO 2018, MIT Lizenz

![image](https://user-images.githubusercontent.com/1107113/37461275-13d85996-284e-11e8-8d1a-74bb3aef82c5.png)

**Comment by frostyandy2k** [comment]
Ah cool, jetzt sehe ich das ScaleIT logo auf dem Browserfenster, nice touch :)

**Comment by laurahalbmann** [comment]
Hier nochmal die überarbeitete PPT: 
[Prototyping Materials_15.03.2018.pptx](https://github.com/ScaleIT-Org/workshop-app-prototyping/files/1815627/Prototyping.Materials_15.03.2018.pptx)

Mit den Logos im Readme benötige ich deine Hilfe :)
